holonote/annotate/table.py

```diff
@@ -58,16 +54,15 @@ def load(self, connector=None, fields_df=None, primary_key_name=None, fields=Non
 
         if fields_df:
             fields_df = fields_df[fields].copy()  # Primary key/index for annotations
-            self._field_df = fields_df
+            self.__field_df = fields_df
```

Comment:
Maybe this should just have a new private name! Whatever you feel is easiest to think about.

Given my suggestion didn't clean up the PR, I'm also happy to revert this and allow `field_df` and `region_df` to be public again. Up to you!

Reply:
For now, let us keep the naming as is.